src/main/java/com/puppycrawl/tools/checkstyle/Main.java

```diff
@@ -602,41 +603,30 @@ private static Properties loadProperties(File file)
     }
 
     /**
-     * Creates the audit listener.
-     *
+     * This method creates in AuditListener an open stream for validation data, it must be closed by
+     * {@link RootModule} (default implementation is {@link Checker}) by calling
+     * {@link AuditListener#auditFinished(AuditEvent)}.
```

Comment:
Please move explanation on why stream is not closed to `getOutputStream(String outputLocation) `.
Unfortunately there is no way to keep reason of suppression in same annotation, so javadoc is only place to keep reason of suppression

Reply:
done